miner/README.org

```diff
@@ -1,9 +1,9 @@
-#+TITLE: Mining for Chainweb
+#+TITLE: Mining for Kadena
```

Comment:
Perhaps "for the Kadena Public Blockchain"?

Reply:
Yes! Much better.